spamclassifier/model_trainer.py

```diff
@@ -1,11 +1,19 @@
 from os.path import dirname as dn, join as jp
 import logging
+import sys
 
 from scipy.io import loadmat
 from sklearn import svm
 import numpy as np
 
 log = logging.getLogger('spamclassifier.model_trainer')
+log.setLevel(logging.DEBUG)
```

Comment:
Setting logging levels and handlers in "framework" code is not a good idea.
Set it in the client code (i.e. test code) if needed, nowhere else.